src/car-utils.js

```diff
@@ -0,0 +1,27 @@
+import Car from "./car.js";
+
+export function makeCarNameList(carNames, separator = ",") {
+  if (typeof carNames !== "string") {
+    throw new Error("자동차 이름은 string 타입이어야 합니다.");
+  }
+  if (carNames.length < 1) {
+    throw new Error("자동차 이름의 길이는 0보다 커야 합니다.");
+  }
```

Comment:
```suggestion
  if (carNames.length > Car.MIN_LENGTH_NAME) {
    throw new Error("자동차 이름의 길이는 0보다 커야 합니다.");
  }
```
MAX_LENGTH_NAME이라는 상수도 있어서 이렇게 MIN_LENGTH_NAME 이라는 상수를 만든 다음에 이렇게 표현해주면 어떨까요!?

Reply:
@JunilHwang 
오 자동차 이름의 최솟값도 상수로 정의하면 좋을 것 같네요! 👍 
2149da09fd226a37d6c8fe48a6e384eb0c78f8f0